tests/notebooks/e2e-wine/e2e-wine-kfp-mlflow-seldon.ipynb

```diff
@@ -0,0 +1,1233 @@
+   },
+   "source": [
+    "## Create Pipeline"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 23,
+   "id": "8dec0d0d",
+   "metadata": {
+    "execution": {
+     "iopub.execute_input": "2023-08-09T14:52:52.062375Z",
+     "iopub.status.busy": "2023-08-09T14:52:52.061403Z",
+     "iopub.status.idle": "2023-08-09T14:52:52.116044Z",
+     "shell.execute_reply": "2023-08-09T14:52:52.115058Z"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "@dsl.pipeline(\n",
+    "    name=\"e2e_wine_pipeline\",\n",
+    "    description=\"E2E Wine Pipeline with MLFlow and Seldon\",\n",
+    ")\n",
+    "def wine_pipeline(url, seldon_deployment_name, model_name):\n",
+    "    web_downloader_task = web_downloader_op(url=url)\n",
+    "    preprocess_task = preprocess_op(file=web_downloader_task.outputs[\"data\"])\n",
+    "    train_task = (\n",
+    "        training_op(file=preprocess_task.outputs[\"output\"])\n",
+    "        .add_env_variable(V1EnvVar(name=\"MLFLOW_TRACKING_URI\", value=os.getenv(\"MLFLOW_TRACKING_URI\")))\n",
+    "        .add_env_variable(V1EnvVar(name=\"MLFLOW_S3_ENDPOINT_URL\", value=os.getenv(\"MLFLOW_S3_ENDPOINT_URL\")))\n",
+    "        .add_env_variable(V1EnvVar(name=\"PROTOCOL_BUFFERS_PYTHON_IMPLEMENTATION\", value=\"python\"))\n",
+    "        .apply(\n",
+    "            use_k8s_secret(\n",
+    "                secret_name=\"mlflow-server-minio-artifact\",\n",
+    "                k8s_secret_key_to_env={\n",
+    "                    \"AWS_ACCESS_KEY_ID\": \"AWS_ACCESS_KEY_ID\",\n",
+    "                    \"AWS_SECRET_ACCESS_KEY\": \"AWS_SECRET_ACCESS_KEY\",\n",
+    "                },\n",
+    "            )\n",
+    "        )\n",
+    "    )\n",
+    "    deploy_task = deploy_op(\n",
+    "        seldon_deployment_name=seldon_deployment_name, model_uri=train_task.output, model_name=model_name\n",
+    "    )"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 24,
+   "id": "7605db80",
+   "metadata": {
+    "execution": {
+     "iopub.execute_input": "2023-08-09T14:52:52.120820Z",
+     "iopub.status.busy": "2023-08-09T14:52:52.120031Z",
+     "iopub.status.idle": "2023-08-09T14:52:52.260324Z",
+     "shell.execute_reply": "2023-08-09T14:52:52.259215Z"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "# local development\n",
+    "kfp.compiler.Compiler().compile(wine_pipeline, \"wine-pipeline.yaml\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 25,
+   "id": "03bb3c27",
+   "metadata": {
+    "execution": {
+     "iopub.execute_input": "2023-08-09T14:52:52.265412Z",
+     "iopub.status.busy": "2023-08-09T14:52:52.264754Z",
+     "iopub.status.idle": "2023-08-09T14:52:52.603976Z",
+     "shell.execute_reply": "2023-08-09T14:52:52.602968Z"
+    }
+   },
+   "outputs": [
+    {
+     "data": {
+      "text/html": [
+       "<a href=\"/pipeline/#/experiments/details/615f01a0-a1a8-498b-b5d6-64bbb454dcac\" target=\"_blank\" >Experiment details</a>."
```

Comment:
I think we can just get rid of all the outputs to not cause confusion (this and "Run details" below return a 404)

Reply:
I'll drop them and only include example outputs in markdown if it helps with the understanding somewhere.